src/YDialogHelpers.cc

```diff
@@ -0,0 +1,233 @@
+/*
+  Copyright (C) 2000-2012 Novell, Inc
+  This library is free software; you can redistribute it and/or modify
+  it under the terms of the GNU Lesser General Public License as
+  published by the Free Software Foundation; either version 2.1 of the
+  License, or (at your option) version 3.0 of the License. This library
+  is distributed in the hope that it will be useful, but WITHOUT ANY
+  WARRANTY; without even the implied warranty of MERCHANTABILITY or
+  FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public
+  License for more details. You should have received a copy of the GNU
+  Lesser General Public License along with this library; if not, write
+  to the Free Software Foundation, Inc., 51 Franklin Street, Fifth
+  Floor, Boston, MA 02110-1301 USA
+*/
+
+
+/*-/
+
+  File:		YDialogHelpers.cc
+
+  Author:	Jiri Srain <jsrain@suse.cz>
+
+/-*/
+
+
+#include <sys/types.h>
+#include <dirent.h>
+#include <fstream>
+
+#include <list>
+#include <algorithm>
```

Comment:
I cannot see the need for the includes above.

Reply:
Fixed.